tests/x11/remote_desktop/vino_screensharing_available.pm

```diff
@@ -56,8 +56,16 @@ sub run {
     }
 
     # Finally ensure that the screen sharing is available
-    assert_screen "with_screensharing";
-    record_info 'vino present', 'Vino and the screen sharing are present';
+    if (check_var('ARCH', 'ppc64le')) {
```

Comment:
ofcourse your version is also make sense .  But I would prefer this one because it is more clear showing what actually happening , that in case of  `ppc64le` you adding extra needle to screen assertion 
```
my @needles = qw(with_screensharing);
push @needles,qw(without_screensharing) if check_var('ARCH','ppc64le');
assert_screen @needles;
```

Reply:
done